packages/vega-util/index.d.ts

```diff
@@ -160,6 +160,6 @@ export const Warn: number;
 export const Info: number;
 export const Debug: number;
 
-export function logger(_?: number, method?: string): LoggerInterface;
-export function log(...args: readonly any[]): void;
+export function logger(_?: number, method?: string, handler?: typeof log): LoggerInterface;
+export function log(method: 'error' | 'warn' | 'log', level: 'error' | 'warn'| 'info' | 'debug', input: readonly any[]): void;
```

Comment:
Should the level be all caps like in https://github.com/vega/vega/pull/3262/files#diff-377e9062f2f35efd117c4c0b89857464b8f54af1c5a8e1e648cc0683b07ca895R24?

Reply:
That is absolutely right, sorry for the confusion